doc/source/ray-debugging.rst

```diff
@@ -0,0 +1,174 @@
+Ray Debugging
+=============
+
+Ray has a built in debugger that allows you to debug your distributed applications.
+
+.. note::
+
+    It is currently an experimental feature and under active development. Interfaces are subject to change.
```

Comment:
Can you please outline the key features? 

I'd like to know what this is/what features this has before I do a code-walkthrough.